Consumer/FirstScenarioRule.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Model;
+
+namespace RabbitMQ.Consumer
+{
+    public static class FirstScenarioRule
+    {
+        /// <summary>
+        /// Check the first rule (Temperature over 59°C + Brightness under 400) 
+        /// </summary>
+        /// <returns></returns>
+        public static bool firtCheckRule()
+        {
+            if (Consumer.consumeTemperature() > 59 && Consumer.consumeBrightness() < 400) ;
```

Comment:
tutto questo codice puoi riassumerlo in un unica riga, senza necessità di "sprecare una variabile". Per intenderci:
public static bool firstCheckRule() 
{
return (Consumer.consumeTemperature() > 59 && Consumer.consumeBrghtness < 400)
}

Così facendo ti tornerà true se entrambe le condizioni sono soddisfate, ovvero in caso di pericolo, altrimenti false. Cerca di cambiarlo a questo metodo e agli altri. Per il resto ottimo!